src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx

```diff
@@ -0,0 +1,268 @@
+import React from "react";
+import { Row, Col, Typography, Form, Input, Button, Select, DatePicker, Radio } from "antd";
+import styles from "./ProfileInfo.module.scss";
+import messages from "../../../../messages";
+
+const { Title, Text } = Typography;
+const { Option } = Select;
+
+function ProfileInfoUI() {
+
+  const config = {
+    rules: [
+      {
+        type: 'object',
+        required: true,
+        message: 'Please select time!',
+      },
+    ],
+  };
+  const selectAfter = (
+    <Select defaultValue="private" className="select-after">
+      <Option value="private">Private</Option>
+      <Option value="public">Public</Option>
+    </Select>
+  );
+
+
+  return (
+    <section className={styles.profileInfo_wrapper}>
+      <Form
```

Comment:
There are multiple `Form`, please split it into its own components

Reply:
Done